x/group/simulation/operations_test.go

```diff
@@ -46,7 +44,6 @@ func (suite *SimTestSuite) SetupTest() {
 		&suite.interfaceRegistry,
 		&suite.accountKeeper,
 		&suite.bankKeeper,
-		&suite.stakingKeeper,
```

Comment:
Why did this change come about, is stakingKeeper just unused in sims in x/groups?

Reply:
Yes, it's not being used anywhere in sims.